static/styles.css

```diff
@@ -277,7 +287,9 @@ body {
   max-height: 30%;
   overflow: auto;
   margin-bottom: 30px;
+  margin-left: 10px;
   margin-right: 30px;
+  margin-top: 20px;
```

Comment:
Instead of writing 4 lines to specify the margin, you can write only one like this: `margin: 20px 30px 30px 10px`. The same for padding.